distribution/docs/src/main/resources/_contents/_core-concepts/metadata-intro-contents.adoc

```diff
@@ -0,0 +1,58 @@
+
+== Understanding Metadata and Metacards
+
+In ${branding}, <<_glossary_resources,resources>> are the data products, files, reports, or documents of interest to users of the system.
+
+Metadata is information about those resources, organized into a schema to make search possible.
+The ${ddf-catalog} stores this metadata and allows access to it.
+Metacards are single instances of metadata, representing a single resource, in the Metadata Catalog (MDC).
+Metacards follow one of several schemas to ensure reliable, accurate, and complete metadata.
+Essentially, Metacards function as containers of metadata.
+
+=== Populating Metacards During Ingest
+
+Upon ingest, a transformer will read the metadata from the ingested file and populate the fields of the metacard.
+Exactly how this is accomplished depends on the origin of the data, but most fields (except id) are imported directly.
+
+=== Searching Metadata
+
+${branding} provides the capability to search the Metadata Catalog (MDC) for metadata.
```

Comment:
I wonder if we need an intro paragraph or two defining what a catalog is? and isn't?